src/consensus/params.h

```diff
@@ -7,8 +7,29 @@
 #define BITCOIN_CONSENSUS_PARAMS_H
 
 #include "uint256.h"
+#include <map>
+#include <string>
 
 namespace Consensus {
+
+enum DeploymentPos
```

Comment:
'position' is a very ambigious, yes. It's almost never a good choice.
Made a similar comment here on another pull: https://github.com/bitcoin/bitcoin/pull/7541#discussion_r53135937
Agree 'Id' seems be a better fit here.